curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-inequality-operator.md

```diff
@@ -23,35 +23,35 @@ The inequality operator (`!=`) is the opposite of the equality operator. It mean
 
 # --instructions--
 
-Add the inequality operator `!=` in the `if` statement so that the function will return "Not Equal" when `val` is not equivalent to `99`
+Add the inequality operator `!=` in the `if` statement so that the function will return `Not Equal` when `val` is not equivalent to `99`
```

Comment:
```suggestion
Add the inequality operator `!=` in the `if` statement so that the function will return the string `Not Equal` when `val` is not equivalent to `99`
```